boms/src/main/resources/bom-template.xml

```diff
@@ -55,6 +55,11 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
+        <groupId>org.wildfly.swarm</groupId>
```

Comment:
What's the need for spi to be directly present in BOM?

Reply:
If a user wants to implement the `ConfigurationFilter` interface they will need to drag in the SPI explicitly, instead of just getting it implicitly/transitively. Hence the need for it to be BOM-able.

Reply:
no problem